src/components/about-offer/AboutOffer.jsx

```diff
@@ -0,0 +1,44 @@
+import PageWrapper from '~/components/page-wrapper/PageWrapper'
+import Typography from '@mui/material/Typography'
+import { useTranslation } from 'react-i18next'
+import aboutOfferTitle from '~/constants/translations/en/common'
+import showMore from '~/constants/translations/en/common'
+import showLess from '~/constants/translations/en/common'
+import { styles } from '~/components/about-offer/AboutOffer.styles'
+import { useState } from 'react'
+
+const AboutOffer = ({ offer }) => {
+  const { t } = useTranslation()
+  const [showFullDescription, setShowFullDescription] = useState(false)
+  const shortDescroption =
+    offer?.description.split(' ').slice(0, 5).join(' ') + ' ...'
+
+  const handleShowMore = () => {
+    setShowFullDescription(true)
+  }
+
+  const handleShowLess = () => {
+    setShowFullDescription(false)
+  }
+
+  return (
+    <PageWrapper>
+      <Typography>{t(`${aboutOfferTitle.aboutOffer}`)}</Typography>
```

Comment:
We can simply specify the key without importing a json:
```javascript
<Typography>{t('common.aboutOffer')}</Typography>
```

Same for the rest translation keys.